script.js

```diff
@@ -14,3 +15,127 @@ window.addEventListener("scroll", function () {
 
   prevScrollPos = currentScrollPos;
 });
+
+//START Form validation
+
+const form = document.querySelector("#form");
+const nameEl = document.querySelector("#name");
+const emailEl = document.querySelector("#email");
+const messageEl = document.querySelector("#message");
```

Comment:
Again, you are jumping into the DOM more than needed. Use `.find()` on the `form` variable.

Reply:
My bad, I meant `form`. I wanted to reduce the times we jump into the DOM.